config_src/drivers/nuopc_cap/mom_surface_forcing_nuopc.F90

```diff
@@ -603,7 +660,8 @@ subroutine convert_IOB_to_fluxes(IOB, fluxes, index_bounds, Time, valid_time, G,
     do j=js,je ; do i=is,ie
       net_FW(i,j) = US%RZ_T_to_kg_m2s * &
                     (((fluxes%lprec(i,j)   + fluxes%fprec(i,j) + fluxes%seaice_melt(i,j)) + &
-                      (fluxes%lrunoff(i,j) + fluxes%frunoff(i,j))) + &
+                      (fluxes%lrunoff(i,j) + fluxes%frunoff(i,j) + &
```

Comment:
Please add parentheses to specify the order of the sum of the 4 run-off variables.  Doing greatly helps with debugging, as it ensures that answers are not overly sensitive to levels of optimization.

Reply:
Addressed in 69220e9cebf8bfdade695c9e617fd6002d99c84a.